students/yeseul/users/views.py

```diff
@@ -41,3 +41,21 @@ def post(self, request):
         
         return JsonResponse({'message' : 'SUCCESS'}, status = 201)
 
+class LoginView(View):
+    def post(self, request):
+        try:
+            request_data = json.loads(request.body)
+            email        = request_data['email']
+            password     = request_data['password']
+        except KeyError:
+            return JsonResponse({'message' : 'KEY_ERROR'}, status = 400)
+
+        try: 
+            User.objects.get(email=email)
+        except ObjectDoesNotExist:
+            return JsonResponse({'message' : 'INVALID_USER'}, status = 401)
+
+        if not User.objects.filter(email=email).filter(password=password).exists():
+            return JsonResponse({'message' : 'INVALID_USER'}, status = 401)
```

Comment:
email 검증과 email, password 검증을 나눠서 할 필요 없이, line 53-56 코드와 line 58-59 코드를 하나로 합치면 좋을 것 같습니다! 

```suggestion
        if not User.objects.filter(email=email, password=password).exists():
            return JsonResponse({'message' : 'INVALID_USER'}, status = 401)
```

Reply:
코드 합쳐서 다시 push했습니다!

Reply:
GOOD